Testscripts/Windows/NTTTCP-MONITOR-THROUGHPUT.ps1

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+param(
+	[String] $TestParams,
+	[object] $AllVMData,
+	[object] $CurrentTestData
+)
+
+function Main {
+	param (
+		$TestParams
+	)
+	$currentTestResult = Create-TestResultObject
+	$scriptFile = "./ntttcp_monitor_throughput.sh"
+	$log_dir = "/root/ntttcp_monitor"
```

Comment:
Can we not use "/root"? Last time, we have clear passdown about this

Reply:
this is not related, as we Run-LinuxCmd as sudo, this is the natural way to use that folder, unless you change Run-LinuxCmd, and change all those codes, where using 'enableRoot.sh', and align with previous test case.  unless you change all previous test cases.